.pre-commit-config.yaml

```diff
@@ -7,14 +7,20 @@ repos:
     rev: "v0.1.1"
     hooks:
       - id: ruff
-        args: [--fix, --ignore=E741, --exclude=__init__.py]
+        args: [--fix, --ignore=E741 , --exclude=__init__.py]
 
-  - repo: https://github.com/nbQA-dev/nbQA
-    rev: 1.7.0
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: v0.1.3
     hooks:
-      - id: nbqa-ruff
-        additional_dependencies: [ruff==0.0.284]
-        args: ["--fix","--ignore=E501,E402"]
+      - id: ruff
+        types_or: [python, pyi, jupyter]
```

Comment:
We already have ruff-pre-commit in place, hence `type_or` should just be added to that section.

Reply:
okay , I will change it quickly.

Reply:
Hello @Saransh-cpp 

Can you please review it again?

- I have removed nbqa_ruff's comments 
-  And fixes this too 
>We already have ruff-pre-commit in place, hence type_or should just be added to that section